core/trino-main/src/main/java/io/trino/sql/planner/plan/AggregationNode.java

```diff
@@ -488,4 +495,90 @@ private void verifyArguments(Step step)
                     arguments.size());
         }
     }
+
+    public static class Builder
```

Comment:
Move
```
    public static Builder builder(AggregationNode node)
    {
        return new Builder(node);
    }
```
above `Builder` class. That's Trino codestyle (see other places)

Reply:
This was based on `Assignments` but I see the other way is more popular